test/gas.js

```diff
@@ -114,6 +114,26 @@ describe("Gas", function() {
             });
       });
     }).timeout(0);
+
+    it("clears mapping storage slots", async () =>{
+      const options = {from: accounts[0]};
+
+      const estimateGasInstance = await deployContract();
+      const uintsa = await estimateGasInstance.methods.uints(1).call();
+      assert.equal(uintsa, "0", "initial value is not correct");
+
+      const receipta = await estimateGasInstance.methods.store(1).send(options);
+      assert.equal(receipta.status, true, "storing value did not work");
+
+      const uintsb = await estimateGasInstance.methods.uints(1).call();
+      assert.equal(uintsb, "1", "set value is incorrect");
+
+      const receiptb = await estimateGasInstance.methods.clear().send(options);
+      assert.equal(receiptb.status, true, "storing 0 did not work");
```

Comment:
This assert message is technically not correcting; `deleting` and `storing` are not the same thing, although `deleting` will resolve in a `0` value when calling it again.

I would just saying `clearing value did not work`